Render breadcrumbs using the govuk_component

klssmith opened the pull request:
- There is no change to the appearance or text of the breadcrumbs, but
they are now rendered by the govuk_component.

[Trello card](https://trello.com/c/TekeTJ9Y/401-use-govuk-component-in-contacts-frontend)

#### Before (full width and mobile width)
![screen shot 2017-01-11 at 12 39 23](https://cloud.githubusercontent.com/assets/12881990/21849154/4648f12c-d7fc-11e6-8d20-03d136006fe9.png)
![screen shot 2017-01-11 at 12 40 04](https://cloud.githubusercontent.com/assets/12881990/21849157/492012a4-d7fc-11e6-9bcc-de4dd73a8ed4.png)

#### After (full width and mobile width)
![screen shot 2017-01-11 at 12 37 18](https://cloud.githubusercontent.com/assets/12881990/21849079/d5e4f0b6-d7fb-11e6-9c51-d509681b5c94.png)
![screen shot 2017-01-11 at 12 37 47](https://cloud.githubusercontent.com/assets/12881990/21849100/f8c695bc-d7fb-11e6-9981-2027095874c7.png)

fofr commented:
Could you include some screenshots of before and after?

Please include mobile widths – we've seen bugs with the breadcrumb component at thin viewports here: https://github.com/alphagov/collections/issues/210

klssmith commented:
Thanks @fofr - I've added the screenshots and changed the commit message to make it clearer that the breadcrumbs should look the same before and after (it's just the way they are rendered that has changed).
